devel/fortio/cmd/echosrv/echo.go

```diff
@@ -23,11 +23,10 @@ import (
 	"flag"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"net/http"
 	"sync/atomic"
 
-	"github.com/golang/glog"
+	log "istio.io/istio/devel/fortio"
```

Comment:
my objection, if you will, here is that you are aliasing a package (that you control) for no reason other than to shorten it and, as a result, we end up with a bit of stutter (`log.LogV()`).

it would be nice if it were a distinct package, but I don't know that it is required.

I notice elsewhere that we don't have the same alias. If anything, I would suggest a consistent approach.